src/pymovements/dataset/dataset_download.py

```diff
@@ -77,6 +77,13 @@ def download_dataset(
     RuntimeError
         If downloading a resource failed for all given mirrors.
     """
+    disclaimer_text = f"""\
+You are downloading the {definition.long_name}. Please be aware that pymovements does not
+host or distribute any dataset resources and only provides a convenient interface to
+download the public dataset resources that were published by their respective authors.
```

Comment:
please also add a second paragraph with the request to cite the paper (the particular citation will be added later so we can be generic):

```
Please cite the referenced publication if you intend to use the dataset in your research.
```

Reply:
done